src/client/src/components/TopNavBar/index.tsx

```diff
@@ -15,27 +15,27 @@ import { isValidNameAndProjectPathSelector } from "../../selectors/wizardSelecti
 
 const messages = defineMessages({
   welcome: {
-    id: "leftSidebar.newProject",
-    defaultMessage: "1. New Project"
+    id: "TopNavBar.newProject",
+    defaultMessage: "New Project"
   },
   frameworks: {
-    id: "leftSidebar.frameworks",
-    defaultMessage: "2. Frameworks"
+    id: "TopNavBar.frameworks",
+    defaultMessage: "Frameworks"
   },
   pages: {
-    id: "leftSidebar.pages",
-    defaultMessage: "3. Pages"
+    id: "TopNavBar.pages",
+    defaultMessage: "Pages"
   },
   services: {
-    id: "leftSidebar.services",
-    defaultMessage: "4. Services (Optional)"
+    id: "TopNavBar.services",
+    defaultMessage: "Add Optional Cloud Services"
   },
   summary: {
-    id: "leftSidebar.summary",
-    defaultMessage: "5. Summary"
+    id: "TopNavBar.summary",
+    defaultMessage: "Summary"
   },
   ariaNavLabel: {
-    id: "leftSideBar.ariaNavLabel",
+    id: "TopNavBar.ariaNavLabel",
```

Comment:
first of all good work 
we should do `topNavBar` for consistency

Reply:
done :)